rcssserver3d/data/rsg/agent/ball.rsg

```diff
@@ -20,8 +20,10 @@
 	 
 	 (setSphereTotal $Mass $Radius)
 	 (nd DragController
-	     (setAngularDrag 0.00005)
-	     (setLinearDrag 0.01)
+	     (setAngularDrag 0.00004)
+	     (setLinearDrag 0.018)
+	     (setStillness 1.5)
+	     (setStillnessBase 3)
```

Comment:
Still about half a dozen things named stillness.

@CarlinWilliamson When this is ready to go, please re-request a review by clicking on the relevant icon 😄 
<img width="208" alt="Screen Shot 2020-09-16 at 8 32 02 pm" src="https://user-images.githubusercontent.com/1217010/93326205-cb047200-f85b-11ea-9f37-cc079216312a.png">

Reply:
haha. oops. fixed now and re-requested